clang/lib/Driver/ToolChains/PS4CPU.cpp

```diff
@@ -152,48 +152,38 @@ void tools::PS4cpu::Linker::ConstructJob(Compilation &C, const JobAction &JA,
     CmdArgs.push_back(Output.getFilename());
   }
 
-  const bool UseLTO = D.isUsingLTO();
   const bool UseJMC =
       Args.hasFlag(options::OPT_fjmc, options::OPT_fno_jmc, false);
 
+  const bool UnifiedLTO = Args.hasFlag(options::OPT_funified_lto,
```

Comment:
This variable is only used once in both functions. You could simplify and just test for the flag in the if expression. Also, it seems like you're testing for `-fno-unified-lto` here, rather than `-funified-lto` due to the default being `true`. You could shorten this a bit by using `hasArg` instead.

Reply:
I believe that `hasArg(options::OPT_fno_unified_lto)` won't have last-one-wins behaviour. For example, `... -fno-unified-lto -funified-lto ...` would result in `UnifiedLTO` being set to `false`, incorrectly.

Reply:
> This variable is only used once in both functions. You could simplify and just test for the flag in the if expression.

I have done this - thanks!